packages/form/addon/lib/field.js

```diff
@@ -801,10 +790,20 @@ export default class Field extends Base {
 
     await this.question.dynamicOptions;
 
-    return value.map((value) => {
-      return validate("inclusion", value, {
-        in: (this.options || []).map(({ slug }) => slug),
-      });
+    return value.map((value) => this._validateOption(value));
+  }
+
+  _validateOption(value, allowBlank = false) {
+    const label = Array.isArray(this.selected)
+      ? this.selected.find((selected) => selected.slug === value)?.label
+      : this.selected?.label;
+
+    return validate("inclusion", value, {
+      in: (this.options || [])
+        .filter((option) => !option.disabled)
+        .map(({ slug }) => slug),
+      allowBlank,
+      label: label ?? value,
```

Comment:
For what is the `label` option in the `validate` function?

Reply:
It's used in the translation